x-pack/solutions/observability/plugins/observability_shared/public/components/add_data_panel/index.tsx

```diff
@@ -78,11 +78,10 @@ export function AddDataPanel({
   onAddData,
   'data-test-subj': dataTestSubj,
 }: AddDataPanelProps) {
-  const { euiTheme } = useEuiTheme();
-  const theme = useTheme();
```

Comment:
@MiriamAparicio I saw this hook here is still using the old theme approach (resulted in an error on the page) As it's a shared one I am not sure if you should change the hook itself or remove it and use everything from `useEuiTheme` but decided to ping you in case you are looking into that.

Reply:
Hi @jennypavlova , was a different issue the one we had, and I fixed here https://github.com/elastic/kibana/pull/207237, I will take a look and open an issue to address everything that uses the old theme on the shared components from logs that we consume 
cc @elastic/obs-ux-logs-team

Reply:
Thank you @MiriamAparicio! It's not a blocker here as I already changed it in our component and left the hook unchanged (as it's a shared one and this PR is fixing something else)